xarray/__init__.py

```diff
@@ -26,6 +26,7 @@
     where,
 )
 from xarray.core.concat import concat
+from xarray.core.coordinates import Coordinates
```

Comment:
Turns out there is a `Coordinate` top-level import from Variable. We should delete that to avoid confusion:

https://github.com/pydata/xarray/blob/a47ff4ed7ff6d6a5a16d0e604bdb120557c42e13/xarray/__init__.py#L38
https://github.com/pydata/xarray/blob/a47ff4ed7ff6d6a5a16d0e604bdb120557c42e13/xarray/core/variable.py#L3142

Reply:
Should we go ahead and delete it without deprecation notice? I'm not sure if anyone is still using `Coordinate`, though, this is quite old.

Reply:
IMO it's ok to delete. It's been so long...

Reply:
Done!